src/server/api/controllers/user.controller.js

```diff
@@ -295,6 +295,24 @@ const editUserRole = async ({ body, id }) => {
       .update(queryData);
   } else return "No edit fields passed, nothing updated!";
 };
+const createAdminBySuperUser = async (id, body) => {
+  const users = await knex
+    .from("users")
+    .select("*")
+    .where({ email: body.email });
+  if (users.length !== 0) {
+    throw new HttpError("Bad request", "user already exists!", 409);
+  }
+  const hashedPassword = await hashPassword(body.password);
+  return knex("users").insert({
+    role_id: 2,
```

Comment:
Instead of hardcoding the id of the admin role, please hardcode the name of the role.

Meaning, make a lookup in the db finding the user role with the name "admin", get the ID and then pass the resulting ID into the insert function. This ensures that id someone changes the order in the seed that makes the roles (or the IDs change in some other way) you don't suddenly end up creating wrong user roles.

See how we did it here: https://github.com/HackYourFuture-CPH/ov-class09-fp/blob/develop/src/server/api/controllers/user.controller.js#L74